_pages/404.md

```diff
@@ -8,7 +8,7 @@ permalink: /404.html
   <div class="cover-container">
     <div class="masthead clearfix">
       <div class="inner">
-        <img src="{{ site.baseurl }}/assets/img/logo-white.svg" width="150" alt="login.gov logo" class='masthead-brand'/>
+        <a href="/"><img src="{{ site.baseurl }}/assets/img/logo-white.svg" width="150" alt="login.gov logo" class='masthead-brand'/></a>
```

Comment:
Linking to "/" will go back to the English homepage. They should go to their language homepage. `{{ site.baseurl }} ` should resolve that.